old/Makefile

```diff
@@ -0,0 +1,48 @@
+GITHUB_API_TOKEN := ""
```

Comment:
please keep the dotfiles in root directory like before. for e.g., `.travis.yml` etc.

and the `old` directory can be deleted.